# This file contains the parameters for the ros_gazebo_gym_panda_example.py script.
ros_gazebo_gym_panda_example_params:
  # ros_ws_abspath: "/home/user/simulation_ws"  # NOTE: Path to the ROS workspace.

  # Q-learning parameters.
  alpha: 0.01     # Learning Rate.
  gamma: 1.0      # Discount factor.
  n_episodes: 2
  n_steps: 250

  # Inference parameters.
  inference: True
  inference_n_episodes: 2
